modules/server/src/network/resolvers/index.ts

```diff
@@ -12,6 +12,12 @@ export type NetworkSearchRoot = {
 	aggregations: Record<string, unknown>;
 };
 
+export type NetworkAggregationArgs = {
+	filters?: object;
+	aggregations_filter_themselves?: boolean;
+	include_missing?: boolean;
+};
```

Comment:
I think a comment block for this would be good. You have a comment below that says "type should match gql typedefs", but it would be good to directly mention which gql typedef, maybe including the path to that file or at least its name. As a first time reader of this file, I should know that this type is describing the inputs to a specific gql query.

Reply:
Good idea! I'll include some kind of name link. I will avoid using path because if files are moved there's a string in a comment that needs to be updated.